ksp_plugin_test/planetarium_test.cpp

```diff
@@ -189,6 +189,43 @@ TEST_F(PlanetariumTest, PlotMethod1) {
   }
 }
 
+TEST_F(PlanetariumTest, PlotMethod2) {
```

Comment:
Based on what you saw with _Mathematica_ it would be good to have a test with a slightly perturbed circle.